src/Components/ScoreBoard/Confetti.tsx

```diff
@@ -0,0 +1,39 @@
+import Player from "../Player/Player";
+
+const numConfetti = 70; // Number of confetti particles to create
+const confettis: {
+  top: string;
+  left: string;
+  backgroundColor: string;
+  animationDelay: string;
+}[] = [];
+const colors = ["#90f1ef", "#ffd6e0", "#ffef9f", "#c1fba4", "#7bf1a8"];
+for (let i = 0; i < numConfetti; i++) {
+  confettis.push({
+    top: Math.random() * 100 + "%",
+    left: Math.random() * 100 + "%",
+    backgroundColor: colors[i % 5],
+    animationDelay: Math.random() * 5 + "s"
+  });
+}
+
+const Confetti: React.FC<{
+  winner?: Player;
```

Comment:
Because not using Player info here, probably better to just pass if there is a winner or not
```suggestion
  winner?: boolean;
```

```suggestion
  show?: boolean;
```

Reply:
fixed